### DISCUSSION TITLE
[v3] .stop does not work.(?)

### DISCUSSION THREAD

**Discussion by romansh** [livewire/livewire#5968] [1. Help] [answered]
Test.php
```php
<?php

namespace App\Livewire;
use Livewire\Component;

class Test extends Component
{

    public function save()
    {
        dd();
    }

    public function doSomething()
    {
    }

    public function render()
    {
        return view('livewire.test');
    }
}
```
livewire.test

```php
<div>
    <form wire:submit="save">
    <button wire:click.stop="doSomething">X111111</button>
        <button type="submit">Save</button>          
    </form>
</div>
```

I expect to get nothing when I click on the button with wire:click.stop="doSomething," but the save() method is called with dd() inside it.

Meanwhile, removing of '.stop' changes nothing.

**Comment by joshhanley** [comment, marked as answer]
@romansh I think what you're after is `.prevent` as the default action of a button is to submit the form, so you want to prevent this from happening. You will also probably want `.stop` to stop it bubbling up the tree 🙂

**Comment by romansh** [comment]
Thank you. You are right, seems I mixed up the actions.

**Reply by joshhanley** [reply]
@romansh no worries! 🙂
